cmd/cluster-agent-cloudfoundry/app/flare.go

```diff
@@ -0,0 +1,131 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
This file appears shouldn't be here, I imagine this was a git merge gone wrong. @DDuongNguyen , can you `git rm` this file and `cmd/cluster-agent-cloudfoundry/subcommands/run/command.go`? 

We'll squash-merge this branch, so no need to try to fix up the old merge or anything like that.